modules/fedimint-mint-client/src/lib.rs

```diff
@@ -1098,8 +1115,26 @@ impl MintClientModule {
         operation_id: OperationId,
     ) -> anyhow::Result<UpdateStreamOrOutcome<ReissueExternalNotesState>> {
         let operation = self.mint_operation(operation_id).await?;
-        let out_point = match operation.meta::<MintOperationMeta>().variant {
-            MintOperationMetaVariant::Reissuance { out_point } => out_point,
+        let (txid, out_points) = match operation.meta::<MintOperationMeta>().variant {
+            MintOperationMetaVariant::Reissuance {
+                legacy_out_point,
+                txid,
+                out_point_indices,
+            } => {
+                // Either txid or legacy_out_point will be present, so we should always
+                // have a source for the txid
+                let txid = txid
+                    .or(legacy_out_point.map(|out_point| out_point.txid))
+                    .context("Empty reissuance not permitted, this should never happen")?;
+
+                let out_points = out_point_indices
```

Comment:
Thought: should we sort indices?

Reply:
They should always be ordered since they are generated from a `Range`:

https://github.com/fedimint/fedimint/blob/209c2f0aca73851faecd0040048a010a405512e1/fedimint-client/src/lib.rs#L1080-L1083